plugins/objectstore/networkservices.go

```diff
@@ -48,6 +49,7 @@ func (n *networkServicesStore) Add(ns *netmesh.NetworkService) {
 		// Not in the store, adding it.
 		n.networkService[key] = ns
 	}
+	logrus.Infof("objectstore: NetworkService was added, %s/%s, total number of elements: %d", ns.Metadata.Namespace, ns.Metadata.Name, len(n.networkService))
```

Comment:
I think we should use the Plugin's logger structure rather than logrus here.

Reply:
right, good catch will change it.

Reply:
Actually these are debug messages. (removed them) logging of event should be done by the plugin which called these methods.